Transport/Smtp.php

```diff
@@ -335,6 +317,11 @@ public function send ( \Hoa\Mail\Message $message ) {
     /**
      * H-MAC.
```

Comment:
This is a well-known cryptographic function, check http://php.net/hash_hmac :-). What would you like to precise?